chroma-manager/chroma_core/plugins/block_devices.py

```diff
@@ -0,0 +1,559 @@
+# Copyright (c) 2017 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+from collections import defaultdict
```

Comment:
Assuming this is mostly gone in a later commit as well.

Reply:
yes almost all